client/src/components/messenger/Chat.js

```diff
@@ -33,7 +33,11 @@ const Chat = () => {
                 {
                     messages.map((msg, key) => {
                         return (
-                            <li key={key} className={"message " + (msg.author ? "author_message" : "friend_message")}>
+                            <li
```

Comment:
the old style is better

Reply:
the old style makes the line too long

Reply:
that makes me a bit sad because it's easier to read.  Lets increase the line length? It seems like this is only long because of indentation...

Reply:
According to ESLint, max line length should be 80 chars, I had it at 90. I just updated it to 120; at that length we should still be able to view an entire line without scrolling

Reply:
I got a big monitor 😄